01/app.js

```diff
@@ -1 +1,17 @@
-console.log('DOM');
\ No newline at end of file
+const commentsItemNewestList = document.querySelectorAll('.comments__item.comments__item--newest'); // uzyłam querySelectorAll by sprawdzić czy nie ma więcej takich elementów z podwójną klasą
+if (commentsItemNewestList) { //chyba nie potrzebne tutaj?
+    commentsItemNewestList.forEach(function (element) {
+        const dataInfoList = element.querySelectorAll('[data-info]');
+        console.log(`liczba znaczników z atrybutem [data-info]: ${dataInfoList.length}`)
+    })
+}
+
+//  ---------Czy lepiej z getElementsByClassName? Szybsze ale starsze rozwiąznaie?---- 
```

Comment:
Może i szybsze, ale przy obecnej szybkości procesorów nie zwracałbym wielkiej uwagi na to.
Dopiero jak zauważymy, że dany element działa wolno to wtedy optymalizujemy.

Reply:
Ok. Czyli rozumiem, ze lepiej z querySelectorAll.